app/editor/src/hooks/api-editor/reports/useApiReports.ts

```diff
@@ -25,23 +25,20 @@ export const useApiReports = (
   const api = useApi(options);
   const download = useDownload(api);
 
-  return React.useMemo(
-    () => ({
-      // Lookups
-      generateCBRAReport: (from: Date, to?: Date | null) => {
-        const params = {
-          from: moment(from).format('YYYY-MM-DDT00:00:00'),
-          to: to ? moment(to).format('YYYY-MM-DDT:11:59:59') : undefined,
-        };
-        return extractResponseData<any>(() =>
-          download({
-            url: `/reports/cbra?${toQueryString(params)}`,
-            method: 'post',
-            fileName: 'cbra.xlsx',
-          }),
-        );
-      },
-    }),
-    [download],
-  );
+  return React.useRef({
```

Comment:
I think I can even get rid of `React.useRef(...)` and just return the object going forward.